index.html

```diff
@@ -3,8 +3,38 @@
     <head>
         <meta name="viewport" content="width=device-width, initial-scale=1" charset="utf-8">
         <link rel="stylesheet" href="stetics.css"> 
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&display=swap" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
     </head>
     <body>
-        <h1>Hello Microverse!</h1>
+        <h1 class="header">
+            <p class="logo">My Logo</p>
+            <div><img class="menu" src="Union.png" alt="MenuBar"></div>
+        </h1>
+        <div class="notheader">
```

Comment:
this is a standalone section of your webpage and I think it is good to use a `section` tag as the parent element instead of a `div` to make the webpage more accessible and the codebase well structured.